File: URI_Python_UFRR/1040.py
linha1 = input().split(' ')
n1, n2, n3, n4 = linha1
n1 = float(n1)
n2 = float(n2)
n3 = float(n3)
n4 = float(n4)
mediaI = ( n1*2 + n2*3 + n3*4 + n4*1 )/10
print('Media: %.1f' %mediaI)
if mediaI >= 7:
	print('Aluno aprovado.') 
elif mediaI < 5:
	print('Aluno reprovado.')
else:
	print('Aluno em exame.')
	n = float(input())
	print('Nota do exame: %.1f'%n)
	mediaII = (n + mediaI)/2
	if mediaII >= 5: print('Aluno aprovado.')
	else: print('Aluno reprovado.')
	print('Media final: %.1f'%mediaII)

File: URI_C_UFRR/1040.c
#include <stdio.h>
int main(){
    float n1,n2,n3,n4,media;
    scanf("%f %f %f %f",&n1,&n2,&n3,&n4);
    media = ( (n1*2) + (n2*3) + (n3*4) + (n4*1) )/10;
    printf("Media: %.1f\n",media);
    if(media>=7){
        printf("Aluno aprovado.\n");
    }
    else if(media<5){
        printf("Aluno reprovado.\n");
    }
    else
    {
        printf("Aluno em exame.\n");
        float n_,media2;
        scanf("%f",&n_);
        printf("Nota do exame: %.1f\n",n_);
        media2 = media;
        media = (media2 + n_)/2;
        if(media2 >= 5){
            printf("Aluno aprovado.\n");
        }
        else{
            printf("Aluno reprovado.\n");
        }
        printf("Media final: %.1f\n",media);
    }
    return 0;
}
